src/sec_newton.mbx

```diff
@@ -558,222 +557,577 @@
         </webwork>
       </exercise>
 
-
-
+<!--Exercise 4 sine function-->
       <exercise>
         <webwork seed="1">
           <setup>
-            <var name="">
-              <static></static>
+            <var name="$f">
+              <static>\sin(x)</static>
+            </var>
+            <var name="$x[0]">
+              <static>1</static>
+            </var>
+            <var name="$x[1]">
+              <static>-0.55740772</static>
+            </var>
+            <var name="$x[2]">
+              <static>0.065936452</static>
+            </var>
+            <var name="$x[3]">
+              <static>-0.000095721919</static>
+            </var>
+            <var name="$x[4]">
+              <static>2.9235662*10^-13</static>
```

Comment:
Yes. The `^` will only grab the next _one_ thing to its right. Except that braces are control characters so if they are used, everything inside the brace pair will be grabbed. 

I do recommend using `10x^{8}`, but I think it is equivalent to `10x^8` when there is only one character to grab.